packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -106,7 +106,10 @@ describe("r2", () => {
 			  -c, --config   Path to .toml configuration file  [string]
 			  -e, --env      Environment to use for operations and .env files  [string]
 			  -h, --help     Show help  [boolean]
-			  -v, --version  Show version number  [boolean]"
+			  -v, --version  Show version number  [boolean]
+
+			Options:
```

Comment:
So what is the difference between an `Option` and a `Flag`? Do we prefer one over the other? I saw that R2's command used options, so I stuck to that